src/metal/interfaces.ts

```diff
@@ -18,6 +18,7 @@ export interface QueueDOMElementInterface {
   id: string;
   callback: (frame: FrameInterface, id: string, bcr: ClientRect) => void;
   el: Element;
+  bcr: ClientRect;
```

Comment:
can we use a more descriptive name? even just rect or clientRect would be great.

Reply:
Agreed. `bcr: ClientRect` is actually used in a few different places within Spaniel. I'll update and include within this PR.